### PR TITLE
Completed home page

### PR DESCRIPTION
**Pull request by mohitdhatrak**
Deploy preview: https://deploy-preview-1--upsport.netlify.app/

Change log: Added the home page of my e-commerce project.

Screenshots - 

image

image

### CODE REVIEW

File: Styles/navigation.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    gap: 0.5em;
+}
+
+.badge-button:hover {
+    background-color: #7dd3fc;
+    border-radius: 0.2em;
+}
+
+.badge-button:focus {
+    outline: var() solid 2px;
```
Comment (nikhil-sharma-nks) [inline Styles/navigation.css:100]: i guess you forgot to add a color in var

**Review by nikhil-sharma-nks — COMMENTED** [review]
looks good to be merged. Nice work👍

File: Styles/index.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+.overlay-text-container {
+    position: absolute;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    /* rgba helped get the background opacity changed to 0.75, opacity property changed opacity of children too */
+    background-color: rgba(54, 54, 54, 0.75);
+    color: #fff;
```
Comment (viru2001) [inline Styles/index.css:30]: set variable for this color as you will use it many times

**Review by viru2001 — COMMENTED** [review]
suggestion: try to reduce CSS if you can

File: Styles/index.css (hunk 2 above)
Comment (mohitdhatrak) [reply Styles/index.css:30]: Actually i have asked vikram a doubt regarding this in that thread, will do after he responds to that

File: Styles/navigation.css (hunk 1 above)
Comment (mohitdhatrak) [reply Styles/navigation.css:100]: Thanks for pointing out

**Comment by mohitdhatrak** [comment]
> suggestion: try to reduce CSS if you can

Thanks for the suggestion, need to use inheritance more i guess.

**Pull request merged**